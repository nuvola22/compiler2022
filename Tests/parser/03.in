begin writeln(3+4-1+2); end.