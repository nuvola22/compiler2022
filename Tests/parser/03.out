      program
         begin
            call
               writeln
               +
                  -
                     +
                        3
                        4
                     1
                  2
